src/build.py

```diff
@@ -705,9 +724,26 @@ def CopyLLVMTools(build_dir, prefix=''):
       CopyLibraryToArchive(os.path.join(build_dir, 'lib', e), prefix)
 
 
+def WinBuildEnv(build_dir, use_gnuwin32=False, bin_subdir=False,
+                runtime='Release'):
+  if not IsWindows():
+    return None
+  cc_env = host_toolchains.SetUpVSEnv(build_dir)
+  if use_gnuwin32:
+    cc_env['PATH'] = cc_env['PATH'] + os.pathsep + os.path.join(GNUWIN32_DIR, 
+                                                                'bin')
+  bin_dir = build_dir if not bin_subdir else os.path.join(build_dir, 'bin')
+  Mkdir(bin_dir)
+  assert runtime in ['Release', 'Debug']
+  host_toolchains.CopyDlls(bin_dir, runtime)
+  return cc_env
+
+
 def LLVM():
   buildbot.Step('LLVM')
   Mkdir(LLVM_OUT_DIR)
+  cc_env = WinBuildEnv(LLVM_OUT_DIR, use_gnuwin32=True, bin_subdir=True)
```

Comment:
This feels weird, because without knowing the implementation of `WinBuildEnv` this looks like `LLVM` assumes it's Windows.

Does it make sense to rename `WinBuildEnv` to `BuildEnv`? Then in theory if we have any non-Win env modifications they go there, and at the callsite it looks more consistent.